docs/beginner/chop-question-4.md

```diff
@@ -0,0 +1,33 @@
+---
+id: chop-question-4
+title: The Chop (Question 4)
+---
+
+import BeginnersGuideProgress from '@site/src/beginnersGuide.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ChopQuestion3 from '@site/image-generator/yml/beginner/chop-question-4.yml';

```

Comment:
Typo, change it to:

`import ChopQuestion4 from '@site/image-generator/yml/beginner/chop-question-4.yml';`